packages/components/src/truncate/types.ts

```diff
@@ -17,19 +29,22 @@ export interface Props {
 	 *
 	 * @default 'auto'
 	 */
-	ellipsizeMode?: TruncateEllisizeMode;
+	ellipsizeMode?: TruncateEllipsizeMode;
 	/**
 	 * Determines the max characters when `truncate` is set.
 	 *
 	 * @default 0
 	 */
 	limit?: number;
 	/**
-	 * Clamps the text content to the specified `numberOfLines`, adding the `ellipsis` at the end.
+	 * Clamps the text content to the specified `numberOfLines`, adding the
+	 * `ellipsis` at the end.
+	 *
+	 * @default 0
 	 */
 	numberOfLines?: number;
```

Comment:
I also realised that this type is not a great fit for `numberOfLines`: according to the [spec](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp), only positive integer numbers make sense (apart from other `string` keywords).

Even a default of `0` is non-sensical